test/jdk/com/sun/crypto/provider/KeyGenerator/Test6227536.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2002, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
As this is a new test, you'll just have the one date:

`...(c) 2022, Oracle and/or`

Reply:
Fixed in: https://github.com/openjdk/jdk/pull/9679/commits/d754211103874b53daf7b8fa9eb276511b40a51b